mpt_trie/src/trie_ops.rs

```diff
@@ -953,11 +995,11 @@ mod tests {
         );
 
         let entries = generate_n_random_fixed_trie_value_entries(MASSIVE_TRIE_SIZE, 0);
-        let big_trie_1 = StandardTrie::from_iter(entries);
+        let big_trie_1 = StandardTrie::try_from_iter(entries).expect("Failed to create trie");
```

Comment:
I might be disagreeing with @muursh a bit here, but if we call `unwrap()`, won't this output the underlying error in the test? Just not a huge fan of the duplicated `expect`s.

Reply:
Nah I'm fine with that being `unwrap()`